sirepo/pkcli/runner.py

```diff
@@ -15,31 +15,33 @@
 import functools
 import os
 from pykern.pkdebug import pkdp, pkdc, pkdlog, pkdexc
+from pykern import pkcollections
 from pykern import pkio
 from pykern import pkjson
-import re
-import shlex
-from sirepo import mpi
 from sirepo import runner_client
 from sirepo import srdb
-from sirepo.template import template_common
+from sirepo.runner_daemon import local_process, docker_process
 import subprocess
 import sys
+import time
 import trio
 
-# How often threaded blocking operations should wake up and check for Trio
-# cancellation
-_CANCEL_POLL_INTERVAL = 1
+# Every _TMP_DIR_CLEANUP_TIME seconds, we scan through the run database, and
+# any directories that are named '*.tmp', and whose mtime is
+# >_TMP_DIR_CLEANUP_TIME in the past, are deleted.
+_TMP_DIR_CLEANUP_TIME = 24 * 60 * 60  # 24 hours
```

Comment:
No comment needed. Use Napoleon/Sphinx/Google `#:` prefix to comments (above) for doc.